shell/app/menus/dop.tsx

```diff
@@ -33,13 +26,13 @@ export const getDopMenu = () => {
       [
         {
           href: goTo.resolve.dopRoot(), // '/dop/projects',
-          icon: <IconApiApp />,
+          icon: <ErdaIcon className="mt-3.5 mr-1" type="api-app" color="currentColor" />,
```

Comment:
`mt-3.5 mr-1`  cannot be set where they are used？contains all menu config

Reply:
modified